test/axes/timeAxisTests.ts

```diff
@@ -303,6 +303,24 @@ describe("TimeAxis", () => {
         assert.strictEqual(annotationFormatter(testDate), "Sun Dec 17, 1995", "formats to a default customized time formatter");
       });
     });
+
+    it("includes the annotation space in the final size calculation", () => {
+      let svg = TestMethods.generateSVG(400, 400);
+      let xScale = new Plottable.Scales.Time();
+      let xAxis = new Plottable.Axes.Time(xScale, "bottom");
+      xAxis.margin(100);
+      xAxis.annotationsEnabled(true);
+      xAxis.annotationTierCount(3);
+
+      xAxis.anchor(svg);
+      xAxis.computeLayout({ x: 0, y: 0}, 400, 400);
+      let coreHeight = xAxis.tickLabelPadding() + xAxis.innerTickLength();
+      let annotationHeight = xAxis.annotationTierCount() * (<any> xAxis)._annotationTierHeight();
+      let minimumHeight = coreHeight + xAxis.margin() + annotationHeight;
+      assert.operator(xAxis.height(), ">=", minimumHeight, "height includes all relevant pieces");
+      xAxis.destroy();
```

Comment:
Should we be doing this going forward? That'd certainly get more testing of `destroy()`.